modules/ROOT/pages/configure-policy-template-source.adoc

```diff
@@ -1,31 +1,33 @@
-= Configuring Template Source for Policies in Runtime
+= Configuring Template Download Location for Policies
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: policy, CloudFront
 
+A policy template, which is a deployable JAR file, contains the business logic for a policy. Each type of policy has a different template based on the rules and logic that needs to implemented.
 
-When an application with a tracked flow is deployed to Mule runtime engine (Mule), the API gateway of that Mule environment communicates with API Manager to request for the applied policies.
+Before a policy is applied to an API, the policy template is downloaded. When an application with applied policies is deployed to a runtime, it requests API Manager for the corresponding policies to be applied to the API.
 
-If the templates for the policies that are to be applied are not already available in Mule, API gateway requests API Manager for the URLs from where to download the template. When you request the template from API Manager, you must specify wether the template source is to be retrieved from a legacy system or a new one.
+//The runtime to which the application is deployed must have API gateway capabilities enabled using the organization or environment credentials where the API instance is defined. Add this content to the How policies work section.
 
+If the policy templates that are to be applied are not already available in the runtime, API gateway requests API Manager for the URLs from where to download the templates.
 
-You configure the following property to provide Policy templates to API Manager from a specified source:
+By default, the legacy system is used as the download location for the templates. You can configure the runtime to download the templates from a specified source using the following system property:
 
 `anypoint.platform.policy_template_source=<source_value>`
 +
 where:
```

Comment:
Do not use "where:" as a transition between a syntax example and value explanations. Use plain text or plain text introducing a list:

"In this syntax, '`external` denotes...and `internal` denotes...

HOWEVER...I don't see either "external" or "internal" in the syntax, so to what does this text apply?

Reply:
Done! Thanks!